Gordon360/Controllers/ProfilesController.cs

```diff
@@ -29,13 +30,16 @@ public class ProfilesController : GordonControllerBase
         private readonly IMembershipService _membershipService;
         private readonly IConfiguration _config;
         private readonly webSQLContext _webSQLContext;
+        private readonly CCTContext _context;
 
-        public ProfilesController(IProfileService profileService, IAccountService accountService, IMembershipService membershipService, IConfiguration config, webSQLContext webSQLContext)
+        public ProfilesController(IProfileService profileService, IAccountService accountService, 
+                                  IMembershipService membershipService, IConfiguration config, CCTContext context, webSQLContext webSQLContext)
```

Comment:
This is the only blocking comment, because changing the argument order after we push could possibly break something.  (I'm not sure, which means I'm not sure it won't.)  So I'd like to be sure the order makes sense, and especially that it won't break some existing code.

Reply:
I think the reason I put `CCTContext` in front of `webSQLContext` is because normally we put `webSQLContext` last because it's a recent addition to the database connection. But I think you're right that it is better to add the new argument (`CCTContext`) from the end.